jib-core/src/main/java/com/google/cloud/tools/jib/filesystem/DirectoryWalker.java

```diff
@@ -31,21 +31,35 @@
 
   private Predicate<Path> pathFilter = path -> true;
 
-  /** Initialize with a root directory to walk. */
+  /**
+   * Initialize with a root directory to walk.
+   *
+   * @param rootDir the root directory.
+   * @throws NotDirectoryException if the root directory is not a directory.
+   */
   public DirectoryWalker(Path rootDir) throws NotDirectoryException {
     if (!Files.isDirectory(rootDir)) {
       throw new NotDirectoryException(rootDir + " is not a directory");
     }
     this.rootDir = rootDir;
   }
 
-  /** Adds a filter to the walked paths. */
+  /**
+   * Adds a filter to the walked paths.
+   *
+   * @param pathFilter the filter.
```

Comment:
`filter returns {@code true} if the path should be accepted and {@code false} otherwise`